### PR TITLE
[IOS-523]: Improve send view fields activating areas

### PR DESCRIPTION
**Pull request by yuliaveres**
image
Reference ☝️  , apart from navigation bar which is not supposed to handle this.

### CODE REVIEW

File: Toshi/Controllers/Wallet/SendToken/SendTokenViewConfigurator.swift [hunk 1]
```diff
@@ -266,10 +290,13 @@ final class SendTokenViewConfigurator: NSObject {
...
 
         let valueContainerView = setupValueContainerView()
+        valueContainerView.addGestureRecognizer(valueEnablingTapGesture)
         stackView.addArrangedSubview(valueContainerView)
         stackView.addSpacing(CGFloat.mediumInterItemSpacing, after: valueContainerView)
         stackView.addStandardBorder(margin: CGFloat.defaultMargin)
-        stackView.addArrangedSubview(setupAddressContainerView())
+        let addressContainetView = setupAddressContainerView()
+        addressContainetView.addGestureRecognizer(addressEnablingTapGesture)
+        stackView.addArrangedSubview(addressContainetView)
```
Comment (designatednerd) [inline Toshi/Controllers/Wallet/SendToken/SendTokenViewConfigurator.swift]: nit: `addressContainerView`

File: Toshi/Controllers/Wallet/SendToken/SendTokenViewConfigurator.swift [hunk 2]
```diff
@@ -623,6 +650,10 @@ extension SendTokenViewConfigurator: UITextViewDelegate {
     func textView(_ textView: UITextView, shouldChangeTextIn range: NSRange, replacementText text: String) -> Bool {
         adjustAddressErrorLabelHidden(to: true)
 
+        if range.length == 0 && text == "\n" {
+            textView.resignFirstResponder()
```
Comment (designatednerd) [inline Toshi/Controllers/Wallet/SendToken/SendTokenViewConfigurator.swift]: I think you should be able to do this returning `false` in `textfieldshouldReturn` after calling this method

Comment (yuliaveres) [reply Toshi/Controllers/Wallet/SendToken/SendTokenViewConfigurator.swift]: it is `textView` 😞

Comment (designatednerd) [reply Toshi/Controllers/Wallet/SendToken/SendTokenViewConfigurator.swift]: aw dammit. 🤦‍♀️

**Pull request merged**